.github/ISSUE_TEMPLATE/-bug_report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Create a bug report
+labels: "template: bug"
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to file a bug report! Please fill out this form as completely as possible.
+
+  - type: dropdown
+    id: packageManager
+    attributes:
+      multiple: true
+      label: What package manager are you using / does the bug impact?
+      options:
+        - npm
+        - yarn 
+        - pnpm
```

Comment:
Not sure it's worth putting this in. They should only be using whatever is recommended by the repo i.e. NPM.